src/hammer-vlsi/hammer_vlsi/driver.py

```diff
@@ -528,6 +530,58 @@ def load_sim_tool(self, run_dir: str = "") -> bool:
             assert isinstance(sim_tool, HammerSimTool)
             return self.set_up_sim_tool(sim_tool, name, run_dir)
 
+    def load_power_tool(self, run_dir: str = "") -> bool:
+        """
+        Loads a power tool on a given database
+
+        :param run_dir: Directory to use for the tool run_dir. Defaults to the run_dir passed in the HammerDriver
+                        constructor.
+        :return: True if power tool loading was successful, False otherwise.
+        """
+        if self.tech is None:
+            self.log.error("Must load technology before loading power tool")
+            return False
+
+        if run_dir == "":
+            run_dir = os.path.join(self.obj_dir, "power-rundir")
+
+        power_tool_name = self.database.get_setting("vlsi.core.power_tool")
+        power_tool_get = load_tool(
+            path=self.database.get_setting("vlsi.core.power_tool_path"),
+            tool_name=power_tool_name
+        )
+        assert isinstance(power_tool_get, HammerPowerTool), "Power tool must be a HammerPowerTool"
+        power_tool = power_tool_get  # type: HammerLVSTool
+        power_tool.name = power_tool_name
+        power_tool.logger = self.log.context("power")
+        power_tool.technology = self.tech
+        power_tool.set_database(self.database)
+        power_tool.submit_command = HammerSubmitCommand.get("power", self.database)
+        power_tool.run_dir = run_dir
+
+        #power_tool.top_module = self.database.get_setting("power.inputs.top_module", nullvalue="")
```

Comment:
Is there a reason these are commented out?

Reply:
Fixed